### PR TITLE
{data}[foss/2020b] nnU-Net v1.7.0 (+ dependencies)

### PR DESCRIPTION
**Pull request by deniskristak**
adding easyconfigs: tifffile-2021.8.30-foss-2020b.eb, unittest2-1.1.0-foss-2020b.eb, SimpleITK-2.1.0-foss-2020b.eb, requests-2.26.0-foss-2020b.eb, batchgenerators-0.23-foss-2020b.eb, Lua-5.3.5-GCCcore-10.2.0.eb

edit (by @boegel): requires ~#14246~ (dependencies for imagecodecs) + ~#14247~ (imagecodecs)

### CODE REVIEW

**Review by Micket — DISMISSED** [review]
A general comment; having to many paths can be a performance problem, so, we often opt to bundle simple things like `requests` into the stuff that needs it rather than having another package with another PYTHONPATH. unittest2, requests, comes to mind here. I'm not saying we have to do this here, but, we need to keep this in mind, because if we never do it, it becomes an increasing problem

File: easybuild/easyconfigs/l/Lua/Lua-5.3.5-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Updated: Denis Kristak
+name = 'Lua'
+version = '5.3.5'
```
[the remark this answers is not in the document]
Comment (deniskristak) [reply easybuild/easyconfigs/l/Lua/Lua-5.3.5-GCCcore-10.2.0.eb:4]: actually yes...when installing with 5.4, we're getting `  Lua version 5.3 is required for SimpleITK_LUA_EXECUTABLE_VERSION.`

**Comment by deniskristak** [comment]
@Micket i think this EC and it's deps are ready to be merged (merge this one last):

https://github.com/easybuilders/easybuild-easyconfigs/pull/14247 -> second to merge
https://github.com/easybuilders/easybuild-easyconfigs/pull/14246 -> first to merge

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node3103.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/d3dbeb8a3c53ab81a0f3257dd2ceec9e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node3518.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/cd3500322b3f291b309175400e34239e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @deniskristak!

**Pull request merged**